packages/web/src/scss/components/Collapse/index.html

```diff
@@ -16,6 +16,8 @@ <h2 class="docs-Heading">Default with button</h2>
   >
     Collapse trigger
   </button>
+  <br />
```

Comment:
I think this should be done using utility classes or using CSS. This is not a good approach, I think.

Reply:
I just deleted the same comment 🙂. I don't insist on it, but I'd prefer a utility class too.

Reply:
I know, but this it is just example, i added spaces because @adamkudrna told me to create nice example, so i did. This is example for Collapse, not for spacing.

Reply:
Ok, i will do it, but i think it is futile